api/models/GameStateRow.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     playedBy: {
       type: 'number',
       isIn: [0, 1],
-      required: true,
+      allowNull: true,
```

Comment:
Same as above. Need to decide if this should be null or 1 for the first gamestate where the cards are dealt

Reply:
Keeping it required and setting it to 1 for the DEAL move as p1 deals